man/Makefile

```diff
@@ -1,10 +1,12 @@
-all: prepare generate
+RUBYBINPATH=$(shell ruby -r rubygems -e 'puts Gem.user_dir')/bin
```

Comment:
with that, `make man` runs under regular user account without access to system Ruby lib directories